### PR TITLE
fix: add Rishav1707

### LINKED ISSUE #1649: [PROFILE ADDITION] <Rishav Raj>
### Profile screenshot

Screenshot (13)

### PR DESCRIPTION
**Pull request by Rishav1707**
resolves #1649 

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

### CODE REVIEW

File: profiles/Rishav1707.json
Comment (Panquesito7) [inline profiles/Rishav1707.json:2]: Should be the provided image in the issue you created.

```suggestion
  "image": "https://user-images.githubusercontent.com/97666287/211783632-12ef84c0-3f6d-4bf8-850c-234981a2e458.png",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Comment by Rishav1707** [comment]
> Thanks! 🎉

When did this PR get merged?

**Comment by Panquesito7** [comment]
> > Thanks! 🎉
> 
> When did this PR get merged? @Panquesito7

Once another maintainer approves the PR, it will get merged. 🙂

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**
